data/json/items/ranged.json

```diff
@@ -4762,6 +4762,36 @@
 "burst": 1,
 "clip_size": 1,
 "reload": 800
+},
+{
+	"id": "heavy_gauss_rifle",
+	"type": "GUN",
+	"symbol": "(",
+	"color": "dark_gray",
+	"name": "heavy gauss rifle",
+	"name_plural": "heavy gauss rifles",
+	"description": "This overpowered and overengineered variant of the ferromagnetic rail rifle was designed to deliver electromagnetically-driven obliteration right through whatever obstacles that lies in front of it, be it buildings, vehicles or hordes of walking dead. However, the hefty bulk and weight of this weapon compared to most contemporary rifles makes it somewhat cumbersome to wield; not to mention that it consumes UPS charges at a tremendous rate, making it impractical for use in prolonged engagements.",
+	"price": 900000,
+	"material": ["steel", "plastic"],
+	"ammo_effects": ["INCENDIARY", "TRAIL"],
+	"ups_charges": 100,
+	"skill": "rifle",
+	"ammo": "metal_rail",
+	"weight": 11500,
+	"volume": 18,
+	"bashing": 12,
+	"cutting": 0,
+	"to_hit": -1,
+	"ranged_damage" : 40,
+	"range" : 40,
+	"dispersion": 100,
+	"sight_dispersion": 150,
+	"aim_speed" : 6,
+	"recoil": 300,
+	"durability": 7,
+	"clip_size": 10,
+	"reload": 300,
+	"valid_mod_locations": [["accessories", 4],["grip", 1],["mechanism", 4],["rail", 1],["sights", 1],["stock", 1],["underbarrel", 1]]
```

Comment:
Mechanism is problematic too, it includes things like "rapid blowback", "auto-fire mechanism", "match trigger".